libcontainerd/remote_linux.go

```diff
@@ -254,6 +256,11 @@ func (r *remote) handleEventStream(events containerd.API_EventsClient) {
 	for {
 		e, err := events.Recv()
 		if err != nil {
+			if strings.Contains(err.Error(), "transport is closing") &&
```

Comment:
Actually looks like you can do `if grpc.ErrorDesc(err) == transport.ErrConClosing.Desc`... probably.